11:59:42 -     INFO - wally           - All info would be stored into /var/wally_results/hobnailed_ariana
11:59:42 -     INFO - wally           - Start discover_stage
11:59:49 -     INFO - wally           - Start reuse_vms_stage
11:59:49 -     INFO - wally           - Start log_nodes_statistic_stage
11:59:49 -     INFO - wally           - Found 7 nodes total
11:59:49 -    DEBUG - wally           - Found 3 nodes with role ceph-mon
11:59:49 -    DEBUG - wally           - Found 3 nodes with role ceph-osd
11:59:49 -    DEBUG - wally           - Found 1 nodes with role testnode
11:59:49 -     INFO - wally           - Start save_nodes_stage
11:59:49 -     INFO - wally           - Start connect_stage
11:59:49 -     INFO - wally           - Connecting to nodes
12:00:04 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:04 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:04 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:04 -    ERROR - paramiko.transport -     self._check_banner()
12:00:04 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:04 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:04 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:04 -    ERROR - paramiko.transport - 
12:00:05 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:05 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:05 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:05 -    ERROR - paramiko.transport -     self._check_banner()
12:00:05 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:05 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:05 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:05 -    ERROR - paramiko.transport - 
12:00:06 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:06 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:06 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:06 -    ERROR - paramiko.transport -     self._check_banner()
12:00:06 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:06 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:06 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:06 -    ERROR - paramiko.transport - 
12:00:06 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:06 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:06 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:06 -    ERROR - paramiko.transport -     self._check_banner()
12:00:06 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:06 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:06 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:06 -    ERROR - paramiko.transport - 
12:00:07 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:07 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:07 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:07 -    ERROR - paramiko.transport -     self._check_banner()
12:00:07 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:07 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:07 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:07 -    ERROR - paramiko.transport - 
12:00:09 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:09 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:09 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:09 -    ERROR - paramiko.transport -     self._check_banner()
12:00:09 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:09 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:09 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:09 -    ERROR - paramiko.transport - 
12:00:12 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:12 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:12 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:12 -    ERROR - paramiko.transport -     self._check_banner()
12:00:12 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:12 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:12 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:12 -    ERROR - paramiko.transport - 
12:00:17 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:17 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:17 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:17 -    ERROR - paramiko.transport -     self._check_banner()
12:00:17 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:17 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:17 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:17 -    ERROR - paramiko.transport - 
12:00:18 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:18 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:18 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:18 -    ERROR - paramiko.transport -     self._check_banner()
12:00:18 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:18 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:18 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:18 -    ERROR - paramiko.transport - 
12:00:19 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:19 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:19 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:19 -    ERROR - paramiko.transport -     self._check_banner()
12:00:19 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:19 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:19 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:19 -    ERROR - paramiko.transport - 
12:00:19 -    ERROR - wally           - During connect to 172.17.3.1: timed out
12:00:19 -    ERROR - wally           - During connect to 172.17.3.2: timed out
12:00:19 -    ERROR - wally           - During connect to 172.17.3.3: timed out
12:00:20 -    ERROR - wally           - During connect to 172.17.1.3: not a valid EC private key file
12:00:21 -    ERROR - paramiko.transport - Exception: Error reading SSH protocol banner
12:00:21 -    ERROR - paramiko.transport - Traceback (most recent call last):
12:00:21 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1710, in run
12:00:21 -    ERROR - paramiko.transport -     self._check_banner()
12:00:21 -    ERROR - paramiko.transport -   File "/usr/local/lib/python2.7/dist-packages/paramiko/transport.py", line 1858, in _check_banner
12:00:21 -    ERROR - paramiko.transport -     raise SSHException('Error reading SSH protocol banner' + str(e))
12:00:21 -    ERROR - paramiko.transport - SSHException: Error reading SSH protocol banner
12:00:21 -    ERROR - paramiko.transport - 
12:00:21 -    ERROR - wally           - During connect to 172.17.1.2: Error reading SSH protocol banner
12:00:23 -  WARNING - wally           - Node(s) 172.17.3.1:22,172.17.3.3:22,172.17.3.2:22,172.17.1.3:22,172.17.1.2:22 would be excluded - can't connect
12:00:23 -     INFO - wally           - Start run_tests_stage
12:00:23 -    DEBUG - wally.sensors   - Installing sensors on 0 nodes
12:00:23 -    DEBUG - wally.sensors   - Starting sensors on 0 nodes
12:00:23 -    DEBUG - wally           - Run preparation
12:00:26 -    ERROR - wally           - File data check is failed - Traceback (most recent call last):
  File "<string>", line 1, in <module>
IOError: [Errno 22] Invalid argument
d41d8cd98f00b204e9800998ecf8427e  -

12:00:26 -     INFO - wally           - Prefilling file /dev/rbd0
12:00:26 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'sudo fio --name=xxx --filename=/dev/rbd0 --direct=1 --bs=4m --size=20480m --rw=write'
12:01:24 -     INFO - wally           - Initiall fio fill bw is 353 MiBps for this vm
12:01:24 -     INFO - wally           - Entire test should takes aroud: 1:20:51 and finished at 13:22:15
12:01:24 -     INFO - wally           - Will run ceph_rws4kth1 test
12:01:24 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:04:54, will wait at most till 12:09:54
12:01:24 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:04:58 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:04:58 -     INFO - wally           - Will run ceph_rws4kth5 test
12:04:58 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:08:28, will wait at most till 12:13:28
12:04:59 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:08:35 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:08:36 -     INFO - wally           - Will run ceph_rws4kth10 test
12:08:36 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:12:06, will wait at most till 12:17:06
12:08:36 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:12:09 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:12:10 -     INFO - wally           - Will run ceph_rws4kth15 test
12:12:10 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:15:40, will wait at most till 12:20:40
12:12:10 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:15:44 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:15:45 -     INFO - wally           - Will run ceph_rws4kth25 test
12:15:45 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:19:15, will wait at most till 12:24:15
12:15:45 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:19:19 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:19:20 -     INFO - wally           - Will run ceph_rws4kth40 test
12:19:20 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:22:50, will wait at most till 12:27:50
12:19:20 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:22:53 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:22:54 -     INFO - wally           - Will run ceph_rrd4kth1 test
12:22:54 -     INFO - wally           - Test should takes about 0:03:30. Should finish at 12:26:24, will wait at most till 12:31:24
12:22:54 -    DEBUG - wally           - SSH:172.17.1.1:22 Exec 'screen -S test -d -m sudo bash /tmp/wally/hobnailed_ariana/io/cmd.sh'
12:23:55 -    DEBUG - wally           - Test on node 172.17.1.1_22 is finished
12:23:55 - CRITICAL - wally           - fio exit with code 128:
